lib/active_set/paginating/operation.rb

```diff
@@ -0,0 +1,102 @@
+# frozen_string_literal: true
+
+class ActiveSet
+  module Paginating
+    class Operation
+      def initialize(set, instructions_hash)
+        @set = set
+        @instructions_hash = instructions_hash
+      end
+
+      def execute
```

Comment:
The logic for this method is to attempt to process all of the instructions with the `ActiveRecordStrategy`, and if that doesn't succeed, to fall back to the `EnumerableStrategy`. 

Paginating as an operation works with "direct" instructions (that is, the instructions don't represent attribute paths or column structures; the instructions hash is a simple, flat hash), and the operation requires all instruction entries together (as opposed to filtering for example, where we can process each instruction entry separately).